AirportTicketBooking/CSVIO/CSVWriterService.cs

```diff
@@ -0,0 +1,19 @@
+using System.IO;
+using CsvHelper;
+using CsvHelper.Configuration;
+
+namespace AirportTicketBooking
+{
+    public class CSVWriterService
+    {
+        public CSVWriterService(string currentDirectory, CsvConfiguration csvConfiguration, string fileName)
+        {
+            var stream = File.Open($@"{currentDirectory}/DataStore/{fileName}.csv", FileMode.Append, FileAccess.Write);
+            StreamWriter = new StreamWriter(stream);
+            CsvWriter = new CsvWriter(StreamWriter, csvConfiguration);
+        }
+        
+        public StreamWriter StreamWriter { get; private set; }
+        public CsvWriter CsvWriter { get; private set; }
```

Comment:
properties should be ubove of the methods 

the file should looks like the following

> properties 
> constructors
> methods

Reply:
Refactored.